index.js

```diff
@@ -162,29 +162,18 @@ async function init() {
 		}
 
 		if (type !== 'tree' || !ref) {
-			return updateStatus(`⚠ ${parsedUrl.pathname} is not a directory.`);
```

Comment:
The error is still correct when the URL points to a blob, e.g. fregante/doma/blob/main/readme.md

The type must be tree or it must be missing